server/controllers/authentication.coffee

```diff
@@ -22,6 +22,8 @@ getEnv = (callback) ->
                 username: username
                 otp:      !!otp
                 apps:     Object.keys require('../lib/router').getRoutes()
+                myAccountsUrl: process.env.COZY_MYACCOUNTS_URL \
+                    or '/app/konnectors/'
```

Comment:
Is the URL `app/konnectors` working on your side (when konnectors is deployed inside the Cozy so)? When I tried it on my own before and with your branch, that doesn't work. I have to use `/apps/konnectors/` instead.